few quick fixes

slr opened the pull request:
there are
* switching to `home.html` page after clicking on desktop notification at pages that do not supporting modal with postboard (such as `network.html`)
* handling of href hash in a second after page loading (in particular to open modal after switching that described above)
* returning of page vertical scrolling after closing of modal window (useful to not lose the locus of attention)
* expanding of posts only by primary button (because there was annoying expanding on clicks on links in posts by middle button in Chromiun based browsers)

slr commented:
* visual changing of follow buttons in dropdown list with search results (and in this case showing of it depending on following status from start of dropdowning) and following suggestions
* updating of postboard in case of following
* updating of followers counter

miguelfreitas commented on js/interface_common.js:
```diff
@@ -544,18 +549,49 @@ var replyInitPopup = function(e, post)
+                            .unbind("click")
+                            .bind("click", userClickFollow);
+
+                        if ($.Options.getTheme() === 'nin') {
```
I'd like to avoid adding any theme-specific javascript code here.
Are you sure we can't have this nin-specific behavior implemented from theme_nin/js/theme_option.js file which is automatically included?

miguelfreitas commented on js/twister_newmsgs.js:
```diff
@@ -101,7 +101,13 @@ function requestMentionsCount() {
         if ($.Options.getShowDesktopNotifMentionsOpt() === 'enable') {
-            $.MAL.showDesktopNotif(false, polyglot.t('You got')+' '+polyglot.t('new_mentions', _newMentions)+'.', false,'twister_notification_new_mentions', $.Options.getShowDesktopNotifMentionsTimerOpt(), openMentionsModal, false)
+            $.MAL.showDesktopNotif(false, polyglot.t('You got')+' '+polyglot.t('new_mentions', _newMentions)+'.', false,'twister_notification_new_mentions', $.Options.getShowDesktopNotifMentionsTimerOpt(), function() {
+                if (window.location.pathname === '/home.html' || window.location.pathname === '/following.html' ) {
```
There must be a better way to tell which current page is instead of relying on absolute filename. This will also probably break support for subdirectory hosting that was recently added I think.

miguelfreitas commented on js/twister_newmsgs.js:
```diff
@@ -101,7 +101,13 @@ function requestMentionsCount() {
+                if (window.location.pathname === '/home.html' || window.location.pathname === '/following.html' ) {
+                     openMentionsModal();
+                } else {
+                    window.location.href = '/home.html#mentions?user='+defaultScreenName;
```
Same here, forcing root /home.html.

slr commented:
* added _$.MAL.showMentions(username)_ and _$.MAL.showDMchat(username)_
  * previous functions binded on clicks on notifications replaced by them
* added follow-unfollow window events and listeners to `home.html` and `following.html`
  * nin-theme related stuff moved into `theme_nin/js/theme_option.js`
* added general prompt template to `following.html`
* added missed notify.js setup to `tmobile.html`

slr commented on js/twister_following.js:
```diff
@@ -353,6 +353,8 @@ function updateFollowing(cbFunc, cbArg) {
 function follow(user, publicFollow, cbFunc, cbArg) {
     if( followingUsers.indexOf(user) < 0 ) {
         followingUsers.push(user);
+        twisterFollowingO.update(user);
```
looks like it covers #195 and another issue that I had previously: I had same account on two nodes with [different](https://twisterio.com/post/tasty/956) number of following users. now it's gone.

slr commented:
* fix of prompt pop-up position in `calm` theme
  * it was scrollable so click on retransmit in middle of postboard was resulting in freeze of window
    * why does click on outside of prompt not close it? is it good?
      * may we put in a project on of these js plugins?
        * https://github.com/chalbert/jQuery-Clickout
        * https://github.com/cowboy/jquery-outside-events
      * I suppose it will be easier to bind'unbind _clickoutside_ event than it is now
* change of header color for prompts in `calm` an `original` themes and modal in `calm` (to get one kind of them)

miguelfreitas commented:
good job!! thanks :)

miguelfreitas commented:
No objections to using these js plugins. 

Btw another nice js plugin i'd love to have time to port from Calm is the one used for top trending filtering based on language/charset... Have you seen that?

Unfortunately my time for developing twister is quite limited lately...

slr commented:
I have heard about that and I have even talked about that [occasionally](https://twisterio.com/post/tasty/1475). many of us want to have that filtering for all posts. I will see a bit later on this.

![git folks are all right](http://31.media.tumblr.com/tumblr_m53neamBi61rwqv34o1_500.gif)
